config/always-pam.m4

```diff
@@ -0,0 +1,36 @@
+AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_PAM], [
+	AC_ARG_ENABLE(pam,
+		AC_HELP_STRING([--enable-pam],
```

Comment:
This ought to be quoted more thoroughly. Also, note it's AS_HELP_STRING.
`AC_ARG_ENABLE([pam], [AS_HELP_STRING([--enable-pam], [text])], ...)`